src/styles/other/variables.scss

```diff
@@ -0,0 +1,19 @@
+$column-width: 100px;
+$column-height: 100px;
+
+$padding-size: 10px;
+
+$minimum-days-in-calendar: 1;
+$maximum-days-in-calendar:31;
+
+$gap-size: 1px;
+
+$week-days: (
+  "mon": 100px,
+  "tue": 200px,
+  "wed": 300px,
+  "thu": 400px,
+  "fri": 500px,
+  "sat": 600px,
+  "sun": 700px,
+);
```

Comment:
What you will do if day-size will change?

Reply:
fixed